src/Search/Kernel.php

```diff
@@ -399,10 +400,10 @@ public function dependencies(Application $app)
                 $config['endpoint'] = $app['config']['aws']['endpoint'];
             }
             if (!isset($app['config']['aws']['credential_file']) || false === $app['config']['aws']['credential_file']) {
-                $config['credentials'] = [
-                    'key' => $app['config']['aws']['key'],
-                    'secret' => $app['config']['aws']['secret'],
-                ];
+                $config['credentials'] = new Credentials(
```

Comment:
If not sure why this is necessary. Will investigate when I can access the app.

Reply:
Reverted changes in last commit. @smoqadam told me that he will include this change in his current branch